packages/server/database/migrations/20230206131155-updateOrgIdOnSAML.ts

```diff
@@ -0,0 +1,16 @@
+import {R} from 'rethinkdb-ts'
+
+export const up = async function (r: R) {
+  await r
+    .table('SAML')
+    .update((row) => ({
+      orgId: row('orgId').default(null)
+    }))
+    .run()
+
+  await r.table('SAML').indexCreate('orgId').run()
+}
+
+export const down = async function (r: R) {
+  await r.table('SAML').indexDrop('orgId').run()
```

Comment:
+1 I think we don't need that migration if we just set a column to null, because rethinkdb is schemaless no need to explicitly add a new column.

I also don't see the index is used in this PR. I suggest to just remove this migration

Reply:
@igorlesnenko,  the index is being used in issue  #7643 to search a `saml` record based on the `orgId`.  however   adding a new optional field `orgId` to the SAML table is part of this PR.  you can  please advice.

Reply:
@gitstart-parabol I see, lets keep the index then

Reply:
Alright @igorlesnenko, the other comments above have been implemented.